### PR TITLE
Apollo portal srv dir

### PR DESCRIPTION
**Pull request by neoformit**
Serve apollo portal from `/srv/sites/apollo_portal` and not `/home/ubuntu`.

### CODE REVIEW

File: ansible/playbooks/playbook-deploy-portal.yml [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
+  - newportalvms
   remote_user: ubuntu
   become: yes
   become_method: sudo
   connection: "ssh"
 
+  vars_files:
+    - group_vars/newportalvms/vars
+    - group_vars/newportalvms/vault
+
```
Comment (neoformit) [inline ansible/playbooks/playbook-deploy-portal.yml:13]: Had to add `vars_files` manually to make it work - I remember having to run the playbook in a certain way to read vars automatically?

Comment (neoformit) [reply ansible/playbooks/playbook-deploy-portal.yml:13]: I ran with `ansible-playbook -i buildnewportalvms.inventory playbook-deploy-portal.yml`

Comment (jlqfab) [reply ansible/playbooks/playbook-deploy-portal.yml:13]: You need to use a build-specific ansible inventory file (`buildnewportalvms.inventory`) and this needs to contain the following to build ubuntu22-test as a django portal
```
[newportalvms]
ubuntu22-test.genome.edu.au  ansible_connection=ssh ansible_user=ubuntu
```
Then run playbook with
`ansible-playbook playbook-deploy-portal.yml --inventory-file buildnewportalvms.inventory --limit ubuntu22-test.genome.edu.au`

File: ansible/playbooks/buildnewportalvms.inventory [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
-ubuntu22-test.genome.edu.au  ansible_connection=ssh  ansible_user=ubuntu
-
 # this is the development portal - django-portal (in hosts)
 apollo-portal-dev.genome.edu.au  ansible_connection=ssh  ansible_user=ubuntu
 
 # this is the production apollo-portal website
 apollo-portal.genome.edu.au  ansible_connection=ssh  ansible_user=ubuntu  portal_extra_domain=genome.edu.au
+
+[testportalvms]
+# ubuntu22-test is for testing deployment only
```
Comment (jlqfab) [inline ansible/playbooks/buildnewportalvms.inventory:9]: ubuntu22 needs to be a member of newportalvms to access the required vars

File: ansible/playbooks/playbook-deploy-portal.yml (hunk 1 above)
Comment (neoformit) [reply ansible/playbooks/playbook-deploy-portal.yml:13]: Awesome - this is the bit I was missing `--limit ubuntu22-test.genome.edu.au`

File: ansible/playbooks/buildnewportalvms.inventory (hunk 2 above)
Comment (neoformit) [reply ansible/playbooks/buildnewportalvms.inventory:9]: Ahh of course

File: ansible/playbooks/playbook-deploy-portal.yml (hunk 1 above)
Comment (jlqfab) [reply ansible/playbooks/playbook-deploy-portal.yml:12]: Can you remove the `vars_files` section - it is not needed.

**Comment by neoformit** [comment]
Also updated the README to document the above (Quick start - how to run a playbook). The rest of the diffs in the README are just adding `sh` to code blocks for syntax highlighting.

**Review by jlqfab — APPROVED** [review]

**Pull request merged**